MATLAB/get_truth.m

```diff
@@ -0,0 +1,236 @@
+function value = get_truth(name,dynamics)
```

Comment:
This requires a corresponding change in the USB console to grab truth from this function rather than from the truth variable, right?

Reply:
Yeah, you can still get the core data directly from dynamics, but it might be nice to let the user request additional data in any frame they want using `get_truth`.